tools/pylib/boutdata/pol_slice.py

```diff
@@ -64,6 +66,43 @@ def pol_slice(var3d, gridfile, n=1, zangle=0.0):
         print("ERROR: pol_slice couldn't read grid file")
         return None
 
+    #Decide if we've asked to do interpolation
+    doInterp = False
+    if nyInterp is not None:
+        if ny != nyInterp:
+            doInterp = True
+
+    #Setup interpolation if requested
+    if doInterp:
+        try:
+            from numpy import linspace, meshgrid, append, newaxis
+        except ImportError:
+            print("ERROR: NumPy module not available")
```

Comment:
I think we require numpy for the pylib stuff, so I think we can just safely import it here